content/docs/platform/inbox/configuration/preferences.mdx

```diff
@@ -0,0 +1,212 @@
+---
+pageTitle: 'Managing Subscriber Preferences'
+title: 'Preferences'
+description: 'Learn how to configure and customize subscriber preferences in your application using the Novu Inbox component.'
+icon: 'UserCog'
+---
+
+The Preferences interface in the Inbox component lets subscribers customize how they receive notifications. By default, Novu renders a preferences icon inside the Inbox component, giving users access to global and workflow-specific settings directly from the UI.
+
+<Callout type = "info">A workflow marked as "critical" on the Novu dashboard cannot be disabled by the user and will not appear in their preferences UI.</Callout>
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/inbox/preferences@2x.png"
+  alt="Preferences"
+/>
+
+## Global preferences
+
+Global preferences apply across all workflows. For instance, a subscriber can disable email or SMS globally ensuring they don’t receive notifications through that channel from any workflow.
+
+![Global preferences](/images/inbox/global-preferences.png)
+
+Global preferences override individual workflow preferences. If a channel is disabled globally, notifications for that channel will not be sent even if it is enabled for a specific workflow.
+
+### Hide Global preferences
+
+Global preferences are always shown in the preferences list and cannot be removed but you can hide it from the UI if needed by adding the following rule to your global CSS file:
+
+```css
+.nv-workflowContainer:first-child {
+  display: none;
+}
+```
+
+## Workflow-specific preferences
+
+Each workflow has its own set of channel preferences in the Inbox UI, where your subscribers can control how they receive notifications for that specific workflow. For example, a user might prefer email for account updates but in-app notifications for security alerts.
+
+![Workflow specific preferences](/images/inbox/workflow-specific-preferences.png)
+
+The Inbox UI only displays the channels used in a given workflow.
+
+<Callout type ="info">Learn how to configure channel steps in a workflow in the [Building Workflows guide](/platform/workflow/build-a-workflow).</Callout>
+
+You can customize how these workflow-level preferences appear to your subscribers.
```

Comment:
@Aviatorscode2 

Could you please share how this works? is this related to filtering preferences section? by default, we show workflow order in which they were created. oldest to newest is the order